README.md

```diff
@@ -43,47 +43,21 @@ project. I did `rbenv install 2.2.3` and then `rbenv local 2.2.3` to use ruby
 Perform this one-time setup:
 1. `bundle install`
 1. `bundle exec berks install`
-1. `vagrant up`
-1. `bundle exec knife solo prepare vagrant@127.0.0.1 -p 2222 -i ./.vagrant/machines/default/virtualbox/private_key`
-   prepares your Vagrant machine for knife-solo development by setting up chef
-   on it.
-1. Run `vagrant snapshot save chef`. If your VM ever becomes inaccessable due to
-   bad chef changes, you can always run `vagrant snapshot restore chef` to reset
-   it to this known-good state.
-
-### Creating a cookbook
-
-Put cookbooks in the `site-cookbooks` folder. `cookbooks` folder is reserved
-for upstream dependencies.
-
-`knife cookbook create -o site-cookbooks/ YOUR_NEW_COOKBOOK`
+1. Start up a node in AWS. There are plugins to do this magically, but :meh:, you might have to
+   create some keys & stuff, so just do this by hand.
+   Confirm that you can login with `ssh ubuntu@12.34.56.78 -i ~/.ssh/leroy_jenkins.pem`.
+1. While that is starting up, go into `data_bags/users/` and create yourself a file, because you'll
+   want to be able to SSH into this box once it's provisioned. Probably.
+1. `bundle exec knife solo prepare ubuntu@12.34.56.78 -i ~/.ssh/leroy_jenkins.pem` This will load
+   all the chef stuff onto it, and give you a file `nodes/12.34.56.78.json`.
 
 ### GO GO GADGET COOKBOOK.
 
-edit nodes/127.0.0.1.json to configure the run_list for your vagrant box.
-
-Put something like `recipe[base]` in there for good measure and let it rip:
-
-`knife solo cook vagrant@127.0.0.1 -p 2222 -i ./.vagrant/machines/default/virtualbox/private_key`
+1. `vi nodes/12.34.56.78.json` and add roles to the run_list. These are the recipes that your box
+   will run, e.g.
+   `["recipe[base]","recipe[factorio]","recipe[factorio::api","recipe[chopped_nginx"]`.
+1. `bundle exec knife solo cook ubuntu@12.34.56.78 -i ~/.ssh/leroy_jenkins.pem` and chef will go off
+   and run all of the cheffy stuff.
 
 On future runs you should be using your own user account to cook and clean the
 box, since the vagrant user may no longer be allowed to SSH.
-
-### Testing chef in Vagrant
```

Comment:
I like the vagrant stuff because it lets you provision a new system to test in like 30 secs.